WPMedia/fusion-news-theme-blocks
# TMEDIA 373 storybook section title

## Description
Storybook for section title

## Jira Ticket
- [TMEDIA-373](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-373&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria
Data for different scenarios with section title are shown 

## Test Steps
- Run `npm run storybook` locally to see section title or use the chromatic checks and review there

## Dependencies or Side Effects


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
